<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>922</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="maximumSize">
    <size>
     <width>800</width>
     <height>559</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="predictBtn">
        <property name="text">
         <string>Predict</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="fileBtn">
        <property name="text">
         <string>File Input</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="3">
       <widget class="QListView" name="starostna_skupina"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Stan</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Statistiƒçna regija</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QListView" name="stan"/>
      </item>
      <item row="1" column="2">
       <widget class="QListView" name="statisticna_regija"/>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Starostna skupina</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Spol</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QListView" name="spol"/>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="predictionLbl">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>922</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
